### ISSUE TITLE
Testpilot site wont load in Fx 57.0a1 (2017-08-08) (64-bit)

### ISSUE THREAD

**Issue by Eeems** [mozilla/testpilot#2746]
When attempting to view any experiments on testpilot.firefox.com I get the following console error. It looks like the site's CSP needs to be updated.
![image](https://user-images.githubusercontent.com/1176979/29091568-a2f0a738-7c40-11e7-9786-e5057465f10d.png)

**Comment by lmorchard** [comment]
Hmm, I'm also on this version of nightly, and I can't reproduce this issue. We have been in the middle of a few deployments to the site today - could be a possible transient issue?

**Comment by stpr-dev** [comment]
I have the same issue with the latest firefox update (55). I visited the site yesterday before the update and everything was fine. 

Here is the console output from today:
```
Content Security Policy: Directive 'frame-src' has been deprecated. Please use directive 'child-src' instead.  (unknown)
Content Security Policy: The page's settings blocked the loading of a resource at data:application/javascript;base64,KGZ1b... ("script-src https://testpilot.firefox.com https://ssl.google-analytics.com").  (unknown)
Loading failed for the <script> with source “https://ssl.google-analytics.com/analytics.js”.  containers:1
Unhandled promise rejection TypeError: f.default.Children.map(...) is null
Stack trace:
value@https://testpilot.firefox.com/static/app/app.e66c374b.js:1:131039
_renderValidatedComponentWithoutOwnerOrContext@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:457906
_renderValidatedComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:458030
performInitialMount@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:453883
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:452929
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:510579
mountChildren@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:506438
_createInitialChildren@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:467842
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:466074
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:510579
mountChildren@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:506438
_createInitialChildren@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:467842
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:466074
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:510579
mountChildren@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:506438
_createInitialChildren@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:467842
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:466074
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:510579
mountChildren@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:506438
_createInitialChildren@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:467842
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:466074
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:510579
mountChildren@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:506438
_createInitialChildren@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:467842
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:466074
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:510579
mountChildren@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:506438
_createInitialChildren@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:467842
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:466074
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:510579
mountChildren@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:506438
_createInitialChildren@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:467842
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:466074
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:510579
mountChildren@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:506438
_createInitialChildren@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:467842
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:466074
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:510579
performInitialMount@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:454047
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:452929
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:510579
mountChildren@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:506438
_createInitialChildren@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:467842
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:466074
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:510579
mountChildren@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:506438
_createInitialChildren@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:467842
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:466074
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:510579
mountChildren@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:506438
_createInitialChildren@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:467842
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:466074
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:510579
performInitialMount@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:454047
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:452929
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:510579
mountChildren@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:506438
_createInitialChildren@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:467842
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:466074
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:510579
performInitialMount@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:454047
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:452929
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:510579
performInitialMount@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:454047
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:452929
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:510579
performInitialMount@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:454047
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:452929
mountComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:510579
_updateRenderedComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:457618
_performComponentUpdate@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:457135
updateComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:456413
receiveComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:455530
receiveComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:510973
_updateRenderedComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:457381
_performComponentUpdate@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:457135
updateComponent@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:456413
performUpdateIfNecessary@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:455746
performUpdateIfNecessary@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:511155
a@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:515608
perform@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:535358
perform@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:535358
perform@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:516700
w@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:516877
closeAll@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:535940
perform@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:535438
batchedUpdates@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:491707
s@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:515872
r@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:513511
enqueueSetState@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:514600
require<[359]</r.prototype.setState@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:571379
require<[341]</n.default/</b</l.prototype.onStateChange@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:554579
f@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:615595
require<["redux-promise"]</n.default/</<@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:919901
setLocalizations@https://testpilot.firefox.com/static/app/app.e66c374b.js:1:176427
value/<@https://testpilot.firefox.com/static/app/app.e66c374b.js:1:173645
T/</<@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:724326
T/<@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:724196
c@https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:684730
  vendor.8b61e861.js:1:386978
TypeError: e is null[Learn More]  vendor.8b61e861.js:1:1
	s https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:1
	r https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:494691
	o https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:494899
	perform https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:535358
	batchedUpdates https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:491707
	batchedUpdates https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:517057
	dispatchEvent https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:495984
	dispatchEvent self-hosted:987:17
	r https://testpilot.firefox.com/static/app/vendor.8b61e861.js:1:392683
```

**Comment by lmorchard** [comment]
Latest Firefox release version I have is v54.0.1 64-bit on Windows. Also have Firefox Developer Edition v55.0b14 (64-bit) on Windows. Can't reproduce this issue on those, either 😖

**Comment by stpr-dev** [comment]
@lmorchard Just to clarify, I can open https://testpilot.firefox.com. However, I cannot open any experiments within (eg: https://testpilot.firefox.com/experiments/containers). 
I tried on both Windows and Linux, same issue. Also, I tried opening on Chromium 60 and same issue.

Is there any way I can get you more information that would help you narrow down the cause of the issue?

**Comment by lmorchard** [comment]
Helpful would be details such as: Add-ons installed? anti-virus running? Seems like some of these errors might be somewhere around google analytics metrics code. We had an issue there, once upon a time, when [uBlock turned off beacon prefs](https://github.com/mozilla/testpilot/issues/2231)

**Comment by stpr-dev** [comment]
- New profile with no addons installed has the same issue. So this is probably not an addon issue. 
- No antivirus installed on my Linux system.

**Comment by stpr-dev** [comment]
`beacon.enable` is set to `true`.

**Comment by jaredhirsch** [comment]
Experiment pages load fine for me. Mac OS 10.12.6, Nightly 57.0a1 (2017-08-08) (64-bit). Also tried on release FF and Chrome.

@collectorgeneral If you're seeing the same problem on multiple machines and browsers, I suspect you might be having a network issue. Can you try disabling any anti-virus or other SSL MITM software or appliances running on your network? Or, could you try tethering off a cell phone connection? I guess it could be some kind of caching issue, too--what's the Last-Modified value returned by, say, `curl -I https://testpilot.firefox.com/experiments/send`? I get "Tue, 08 Aug 2017 19:55:27 GMT".

@clouserw Do the GA stats for experiment page visits show any kind of drop in traffic today?

As for CSP errors, it looks like there's a bug in today's Nightly where every CSP error from every loaded page shows up in the console when a page is loaded. So, on the Test Pilot site, I see a bunch of CSP errors in the console, including a few for the Test Pilot site, but all of the experiment pages load fine for me.

CSP errors screenshot:

<img width="1373" alt="screen shot 2017-08-08 at 2 06 48 pm" src="https://user-images.githubusercontent.com/96396/29094701-1cd58760-7c43-11e7-8e71-ecb42d272e22.png">

**Comment by jaredhirsch** [comment]
Experiment pages load fine for me on Windows 10, both release channel and today's nightly.

**Comment by stpr-dev** [comment]
@6a68 Curl output is the same as what you have: `last-modified: Tue, 08 Aug 2017 19:55:27 GMT`
The machines I tried are not on the same network. I even tried it on my phone with my mobile network. Same outcome.

I have however found that the page loads on `google-chrome` (**NOT** `chromium`). Perhaps this has something to do with google services?

**Comment by stpr-dev** [comment]
To further add to the confusion: the websites loads on `palemoon`. The only thing on the console was:
`21:35:51.339 mutating the [[Prototype]] of an object will cause your code to run very slowly; instead create the object with the correct initial [[Prototype]] value using Object.create1 vendor.8b61e861.js:1:829966`

**Comment by Eeems** [comment]
I've restarted my browser in safe mode, with CSP disabled. I'm still getting the unhandled promise rejection and type error. Seems to me like it's something other then a problem with the latest version of Fx Nightly. (I tested on nightly 56 as I hadn't updated on one of my machines yet).
Last-Modified is Tue, 08 Aug 2017 19:55:27 GMT for me as well.

**Edit:**
I've just tested with Firefox 54 (32-bit) and also had the exact same problem.
After creating a brand new profile it starts working again. I'm trying to track down just what is causing the problem.

**Comment by jaredhirsch** [comment]
@Eeems What addons do you have installed in the profile that's having the problems?

**Comment by Eeems** [comment]
@6a68 Well since I've attempted in safe-mode I doubt it's the addons.
Raw troubleshooting info: https://ghostbin.com/paste/oyv57
Potentially something privacy settings touches though as changed flags will survive the addon being disabled?

**Comment by stpr-dev** [comment]
Here is an update. So I downloaded the [Firefox Portable](https://portableapps.com/apps/internet/firefox_portable) and tried accessing one of the experiments and I am getting them to open just fine. I am really confused because I have tried the same on an installed version on both Windows and Linux, with completely new profiles and visiting the site immediately upon opening the profiles and I cannot get to access the webpage whatsoever. This is really baffling.

**Comment by Eeems** [comment]
@collectorgeneral Now that's really strange. I'm at least able to visit the pages when using a new profile. Mind checking the troubleshooting information on that new profile and comparing it to mine to see if there are similarities?

**Edit:** as a side note, I've tried re-installing testpilot to make sure I have the latest version. Still no dice.

**Edit2:** I've done a little debugging into what it's actually erroring on. It seems that the actual error has been abstracted away. As far as I can tell though the load event on img.experiment-icon is throwing an exception, but when the exception is caught by a try/catch to be thrown, the variable it's suppose to be stored in to be thrown with is null or undefined. [/frontend/src/app/containers/ExperimentPage.js#L352](https://github.com/mozilla/testpilot/blob/44653b678775501ff7774c9cff7528df05c344e9/frontend/src/app/containers/ExperimentPage.js#L352) seems to be the image in question, but I'm not seeing any load events. This is probably abstracted away by React.

**Comment by Eeems** [comment]
Based on the number of duplicate cases I'm going to assume quite a few people are affected by this. Has anybody been able to recreate this based on the troubleshooting information I've provided?

**Comment by stpr-dev** [comment]
@Eeems I do not have access to my Windows machine at the moment for the portable profile. 
My current system is Linux based, and [here](https://ghostbin.com/paste/5rg4r) is a troubleshoot information from the browser using a new profile (no addons, nothing changed). I have been trying to experiment with different versions including the older 54 and ESR releases, to no avail.

**Comment by toyg** [comment]
I have similar symptoms in #2749 , the looping error in console however is:

     BindInt32Parameter is deprecated and will be removed soon.  (unknown)
     Sending message that cannot be cloned. Are you trying to send an XPCOM object?   
            addonManager.js:206:10

**Comment by Eeems** [comment]
@collectorgeneral Based on the other related issues, I don't think it's a Windows specific issue. I'm also not sure it's a problem with a Nightly build in particular. You may want to have your test profile emulate my changed preferences to see if it's one of those.

@toyg It may well be related. The error message is different, but they may both be symptoms of the same issue. I don't know enough about what everything is doing to know for sure though.

**Comment by stpr-dev** [comment]
@Eeems I had tried changing settings that I found from your Troubleshoot log in a new profile in about:config tab (where applicable), to no avail.

**Comment by chuckharmston** [comment]
We very much appreciate all the help narrowing this down, everyone! @lmorchard and I spent much of today trying to reproduce this to no avail, so any additional information helping us get there is great.

My current working theory is that this is an interaction with _something_ in older Firefox profiles; all of what I'm able to test on Windows or Linux right now are new ones. If anyone has a profile where this is happening and wouldn't mind zipping it up and sharing with me, that would be most helpful.

`chuck@m.c`

**Comment by davehunt** [comment]
I've hit this today on my new laptop. It arrived yesterday, and I used Firefox sync with a new profile, which appears to have brought across containers, which I was using on my previous laptop. When visiting https://testpilot.firefox.com/ the experiment does not indicate that it is active, and when clicking on 'Get started' the experiment page never loads.

<img width="1115" alt="screen shot 2017-08-11 at 14 06 02" src="https://user-images.githubusercontent.com/122800/29214193-3b19fcd0-7e9e-11e7-80d1-af28d0eb2511.png">

**Comment by Eeems** [comment]
@davehunt Interesting. I use to have containers active, but I don't have it active anymore. I wonder if it did something to my profile that has caused this.

@chuckharmston I can probably send you my profile sometime later today. Any preference on how I send it?

**Comment by SoftVision-PaulOiegas** [comment]
Hi @Eeems, I'm one of the QA's from Test Pilot. Sending a profile that reproduces the issue would be great. Just archive it and send it via email if it contains sensitive data (we can provide some addresses). Or if it's not a problem to share it publicly, you can upload the profile in a cloud service (archived or not) and comment here with a link to it.

My address is soche2009@gmail.com. I will make sure it will reach the development team also.

**Comment by toyg** [comment]
[testprofile.zip](https://github.com/mozilla/testpilot/files/1218377/testprofile.zip)

Blank profile exhibiting the behaviour on OSX.

**Comment by lmorchard** [comment]
I just merged @fzzzy's PR #2766 - that *might* be the fix to this. Still can't reproduce this myself, but ([in about 10 minutes if CircleCI cooperates](https://circleci.com/gh/mozilla/testpilot/5343)) you can visit https://testpilot.dev.mozaws.net/ to see the current dev build from master branch.

Problem with that, though, is that you have to run through [our development quickstart setup](https://github.com/mozilla/testpilot/blob/master/docs/development/quickstart.md) to actually see content on the site. (Just the parts in Firefox, not the parts that involve installing node & cloning the git repo & building the site.) That's a lot to ask, but if anyone who can reproduce this problem feels like doing that and browsing the dev site, that would be awesome.

Of course, it *could* be the development setup that makes me unable to reproduce the issue in the first place so `¯\_(ツ)_/¯`

**Comment by Eeems** [comment]
@lmorchard I've just shared my profile with @SoftVision-PaulOiegas so you may be able to recreate it with that.

**Comment by lmorchard** [comment]
Yay, I finally have steps to reproduce:

* Go to language options
* Delete my default of en-US
* Add en-CA and make that the default

And now I see the error. Curse my US-centric ways!

**Comment by Eeems** [comment]
@lmorchard It's funny/sad how many things I've broken due to not having en-US in my language list. I'm glad we found the problem!

**Comment by stpr-dev** [comment]
@lmorchard Holy smokes yes! My language was en-GB and when I added and moved en-US up the list it works! 
No wonder the portable apps version worked, it defaults to en-US. Curse US centric indeed 😄

**Comment by Eeems** [comment]
@collectorgeneral I've added en-US to the bottom of my list and that also fixes it.

**Comment by lmorchard** [comment]
@eeems @toyg Thanks for your profile sharing! That was the piece we were missing

**Comment by chuckharmston** [comment]
For what it's worth, the issue is likely not that it's missing `en-US`; it's that it includes a locale that we do not have translations before (`en-CA` or `en-GB` being two).

Thanks for the help hunting this down, everyone!

**Comment by stpr-dev** [comment]
@Eeems Yup, just tried that, that works too. Phew!

**Comment by Eeems** [comment]
@chuckharmston and from what I hear including `en` in the list wont work as it's broken.

**Comment by SoftVision-PaulOiegas** [comment]
@Eeems Yes, for the moment, you all can add [en-us] in the list at the bottom, so the browser will fallback to that. This until we fix the [en] localization.

Anyway, congrats @SoftVision-CosminMuntean for finding the cause !!! 👍

**Comment by stpr-dev** [comment]
@Eeems That is correct, en was included in my current profile along with en-GB and en-CA and it did not work. Only after adding en-US it works.

**Comment by lmorchard** [comment]
So, I can reproduce on production. The error looks fixed on dev after merging #2766 - though there are some spots where strings are blank.

Also: I can turn the error on & off locally by reverting & reapplying #2766 - so I think we're onto something

**Comment by SoftVision-PaulOiegas** [comment]
@lmorchard I can confirm also that the main issue is already fixed on Dev server with that PR. Just compared it with stage on same profile.

**Comment by grahamperrin** [comment]
## SeaMonkey 2.4.8 on FreeBSD-CURRENT

All of https://testpilot.firefox.com/ https://testpilot.firefox.com/experiments and https://testpilot.firefox.com/experiments/tab-center are bugged, I assume that other pages are similarly affected. 

2017-08-13 02:39:54

![2017-08-13 02 39 54 firefox test pilot in seamonkey](https://user-images.githubusercontent.com/192271/29393609-acdae7c8-82fc-11e7-8494-0cdf7dacf22d.png)

Whilst I don't expect a Firefox Test Pilot experimental `.xpi` to work in SeaMonkey, I do expect basic information about Test Pilot to be legible across multiple browsers. 

## Firefox 55.0.1 on FreeBSD-CURRENT

https://testpilot.firefox.com/experiments/send

![2017-08-13 11 37 16 firefox test pilot in firefox](https://user-images.githubusercontent.com/192271/29393530-2440e890-82fc-11e7-9047-70c7e3dce57e.png)

## Not bugged

URLs such as https://testpilot.firefox.com/experiments/send with Chromium 59.0.3071.115 (Developer Build) (64-bit) on FreeBSD-CURRENT. 

## Maybe a duplicate of this issue

[Test Pilot website doesn't load on Firefox on Android · Issue #2788 · mozilla/testpilot](https://github.com/mozilla/testpilot/issues/2788)

**Comment by grahamperrin** [comment]
## Internet Archive Wayback Machine

http://web.archive.org/web/20170718085648/https://testpilot.firefox.com/experiments/tab-center (bouncing ball) http://web.archive.org/web/20170812084521/https://testpilot.firefox.com/ (fallen hamburger) and so on. 

## Effectiveness of workarounds

> … Only after adding en-US it works.

In Seamonkey I found en-US primary, en secondary, nothing else; **not** a workaround, unfortunately. 

I removed en, leaving en-US alone. Control-F5, restarted the browser, still no go. Sorry.

**Comment by Eeems** [comment]
@grahamperrin Have you checked the console to see if it's the same error or a new one?
I'm not able to replicate the problem in 55.0.2 (32-bit) on windows 10.

**Comment by lmorchard** [comment]
We just deployed what should be fixes to this issue today

**Comment by grahamperrin** [comment]
https://testpilot.firefox.com/experiments/send is OK now (no longer requires the workaround) in Firefox 55.0.2. 

Various URLs still bugged in the Wayback Machine e.g. five minutes ago https://web.archive.org/web/20170817174528/https://testpilot.firefox.com/experiments and in Seamonkey – should I raise a new issue now, or wait a few hours?

**Comment by lmorchard** [comment]
> Various URLs still bugged in the Wayback Machine 

Not sure what you're testing there... We can't control content in the wayback machine, and it's probably the site from before our last deployment - which would contain the buggy code.

**Comment by grahamperrin** [comment]
> … from before our last deployment … 

The capture was made just over twenty minutes ago.

**Comment by Eeems** [comment]
@grahamperrin as @lmorchard said, they don't control the content. If the live version is fixed, that should be enough.
